src/renderer/component/walletSend/view.jsx

```diff
@@ -56,8 +56,10 @@ class WalletSend extends React.PureComponent<Props> {
                     label={__('Amount')}
                     postfix={__('LBC')}
                     className="input--price-amount"
+                    fixClass="form-field--fix-no-height"
                     min="0"
                     step="any"
+                    placeholder="5"
```

Comment:
Sometimes placeholders can cause confusion among users who think the placeholder values are actually in the input.

While styling is the primary way to handle this, placeholder values that are less likely to be real values can also help.

(E.g. `12.34` may be a better cue than `5`)